src/goto-programs/goto_program.cpp

```diff
@@ -32,51 +37,80 @@ std::ostream& goto_programt::output_instruction(
   std::ostream& out,
   instructionst::const_iterator it) const
 {
-  out << "        // " << it->location_number << " ";
+  return output_instruction(ns, identifier, out, *it);
```

Comment:
You might want to come up with a brief work estimate on the number of changes that removing this function induces, but I'd encourage getting rid of it.